.github/workflows/node.js.yaml

```diff
@@ -0,0 +1,24 @@
+name: Node.js CI
+
+on: [push, pull_request]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [16.x]
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v2
+        with:
+          node-version: ${{ matrix.node-version }}
+      - run: npm ci
+        working-directory: server
+      - run: ./node_modules/.bin/eslint .
```

Comment:
also run tests? or do you want to add that line when we merge the next PR? @pcgamer1

Reply:
Yes, in the next one.